packages/react-ui-core/src/Photo/Photo.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+
+@themed(['Photo'], {
+  pure: true,
+})
+
+export default class Photo extends PureComponent {
+  static propTypes = {
+    url: PropTypes.string,
+    theme: PropTypes.object,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const { theme, className, url, ...rest } = this.props
+
+    return (
+      <img
+        src={url}
```

Comment:
If we use a fallbackUrl like above, we could use onError here to accomplish it in the image